plugin/src/main/kotlin/com/strumenta/kolasu/languageserver/plugin/LanguageServerPlugin.kt

```diff
@@ -185,16 +186,23 @@ class LanguageServerPlugin : Plugin<Project?> {
         } else {
             var grammars = ""
             if (Files.exists(configuration.textmateGrammarPath)) {
```

Comment:
Consider introducing a variable for `configuration.textmateGrammarPath` and replace usages.
If this is applied, consider removing variable filename in the following line